src/pages/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/troubleshooting.md

```diff
@@ -62,3 +70,43 @@ If you are using the default NFS storage however, then the script pod would be d
 
 - being evicted due to exceeding its storage quota
 - being moved or restarted as part of routine cluster operation
+
+## Frequently Asked Questions {#FAQ}
+
+### Can the agent work with Octopus running in an HA Cluster setup?
+Yes! See the [Kubernetes agent HA Cluster Support](/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/ha-cluster-support) page.
+
+
+### Can a proxy be specified when setting up the Kubernetes agent? 
+Yes! Proxy servers for the polling connection that takes place between the agent and Octopus Server. These can be supplied for setup via the `.pollingProxy.*`  helm values.
+
+Define the polling proxy server through the `agent.pollingProxy.host`, `agent.pollingProxy.port`, `agent.pollingProxy.username` and `agent.pollingProxy.password` values via the [octopusdeploy/kubernetes-agent](https://hub.docker.com/r/octopusdeploy/kubernetes-agent) helm chart.
+
+### When trying to install the Kubernetes Agent on an existing cluster, I get an 401: Unauthorized error.
+
+```
+Error: GET "https://registry-1.docker.io/v2/octopusdeploy/kubernetes-agent/tags/list":
```

Comment:
this is based on this exchange:

![image](https://github.com/user-attachments/assets/d84d0ec7-08a4-4e70-977b-adf5fc315c77)